scripts/macos_setup_exiv2.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+EXIV2_BUILD_NAME="exiv2-0.27.2-Darwin"
+EXIV2_LIB_NAME="libexiv2.0.27.2.dylib"
+EXIV2_LIB_RELEASE_NAME="libexiv2.27.dylib"
+
+mkdir -p release/exiv2
+cd release
```

Comment:
napewno nie chcesz tego setupować w targecie?

Reply:
W sumie to nie wiem. Założyłem, że target jest typowo Cargowy i żeby tam nie mieszać. Zrobię rozeznanie i zobaczę jak się sprawa przedstawia.

Reply:
ja upierał się nie będę, prawda, że może być to tricki